cdc/cdc.cc

```diff
@@ -64,12 +65,116 @@ using namespace std::chrono_literals;
 
 static logging::logger cdc_log("cdc");
 
-class cdc::cdc_service::impl {
+namespace cdc {
+static schema_ptr create_log_schema(const schema&, std::optional<utils::UUID> = {});
+static schema_ptr create_stream_description_table_schema(const schema&, std::optional<utils::UUID> = {});
+static future<> populate_desc(db_context ctx, const schema& s);
+}
+
+class cdc::cdc_service::impl : service::migration_listener::empty_listener {
```

Comment:
I would like to see a textual description of what this service does, which replaces the documentation that `cdc::setup` and `cdc::remove` functions previously had.

Reply:
Sure.